File: GO/1007.go
package main

import "fmt"

func main() {
	var a, b, c, d int
	fmt.Scanln(&a)
	fmt.Scanln(&b)
	fmt.Scanln(&c)
	fmt.Scanln(&d)
	fmt.Printf("DIFERENCA = %d\n", ((a*b)-(c*d)))
}

File: JavaScript/1007.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var firstInput = parseInt(lines.shift());
var secondInput = parseInt(lines.shift());
var thirdInput = parseInt(lines.shift());
var fourthInput = parseInt(lines.shift());

var diferenca  = (firstInput*secondInput) - (thirdInput*fourthInput);

console.log("DIFERENCA = " + diferenca);
